cypress/visual/AppNavigationItem.cy.ts

```diff
@@ -0,0 +1,36 @@
+import { mount } from 'cypress/vue2'
+import Vue, { defineComponent } from 'vue'
+import VueRouter from 'vue-router'
+
+import NcAppNavigationItem from '../../src/components/NcAppNavigationItem/NcAppNavigationItem.vue'
+
+describe('NcAppNavigationItem', () => {
+	it('With router link is primary on active', () => {
```

Comment:
`it With router link is primary on active` sounds weird for me =D

If the `spec` style is not used, maybe use `test` style?

Reply:
yes, makes sense :)